.eslintrc.json

```diff
@@ -54,5 +52,6 @@
     "pickRandomsFromList": true,
     "respecConfig": true
   },
+  "ignorePatterns": ["src/core/text-loader.js"],
```

Comment:
Waiting on https://github.com/eslint/eslint/pull/13196. Can get rest of the PR in before that though, but no hurries.